src/scripts/main.js

```diff
@@ -1,3 +1,259 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+let sortTargetInnerText = '';
+let sortDirection = 1;
```

Comment:
what means 1? lets avoid using magic numbers system and use understandable constants

Reply:
This is a change of sort direction.
When multiplying by 1, it is asc sorted
When multiplying by -1, it is desc sorted